.github/github-bot.yml

```diff
@@ -19,4 +19,6 @@ github-team:
 prchecklist:
   title: Pull Request Checklist
   checklist:
-    - 'Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?'
+    - 'Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/build_status/))?'
+    - 'Have you written tests for new/changed app behavior?'
+    - 'Do new UI items contain Automation IDs?'
```

Comment:
I would rather change it to "Do all new elements contain accessibility IDs?"
And also added:
- Please, describe briefly what you tested and what platforms you used.